applier: fix subscribe_request::id_filter initialization (#11060)
tarantool/tarantool: changelogs/unreleased/gh-11156-fix-calculation-of-sync-quorum-in-config-and-supervised.md

## bugfix/replication

> * Fixed a bug when the node configured as
>   `box.cfg.bootstrap_mode` = `'config'`/`'supervised''` didn't go into an

Reviewer: ```suggestion
* Fixed a bug when a node configured as
  `box.cfg.bootstrap_mode` = `'config'`/`'supervised''` didn't switch to the
```

Author: Applied.
